### PR TITLE
allow specifying ctlog log prefix in fulcio.

### PR DESCRIPTION
**Pull request by k4leung4**
Signed-off-by: Kenny Leung <kleung@chainguard.dev>

 **Description of the change**

allow specifying ctlog log prefix in fulcio.

also bump up ctlog version.

 **Existing or Associated Issue(s)**

 **Additional Information**

 

**Checklist**

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). Where applicable, update and bump the versions in any associated umbrella chart
- [x] Variables are documented in the `values.yaml` and added to the README.md. THe [helm-docs](https://github.com/norwoodj/helm-docs) utility can be used to generate the necessary content. Use `helm-docs --dry-run` to preview the content
- [x] JSON Schema generated
- [x] List tests pass for Chart using the [Chart Testing](https://github.com/helm/chart-testing) tool and the `ct lint` command

### CODE REVIEW

**Review by sabre1041 — CHANGES_REQUESTED** [review]
@k4leung4 Can you generate a new JSON schema?

File: charts/fulcio/values.yaml [hunk 1]
```diff
@@ -93,6 +93,7 @@ ctlog:
   name: ctlog
   forceNamespace: ctlog-system
   fullnameOverride: ctlog
+  logPrefix: sigstorescaffolding
```
Comment (hectorj2f) [inline charts/fulcio/values.yaml:96]: Shouldn't it be `logPrefix=fulcio` here ?

Comment (hectorj2f) [reply charts/fulcio/values.yaml:96]: Someone might deploy fulcio without using scaffolding.

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**